styles.css

```diff
@@ -0,0 +1,542 @@
+body {
+  box-sizing: border-box;
+  margin: 0 90px;
+  border: 1px solid rgb(241, 231, 231);
+  overflow-x: hidden;
+}
+
+a {
+  color: black;
+}
+
+.nav-color {
+  background: rgb(250, 0, 28);
+  background: rgb(247, 34, 16);
+}
+
+.botton-radius {
+  border-radius: 20px;
+}
+
+.nav-border {
+  border-right: 1px solid #d9d9d9;
+}
+
+.nav-2 {
+  border-top: 1px solid #d9d9d9;
+  border-bottom: 1px solid #d9d9d9;
+}
+
+.logo-img-lg {
+  width: 500px;
+}
+
+.bg-orange {
+  background-color: rgb(247, 34, 16);
+}
+
+.btn-orange {
+  color: white;
+  background-color: rgb(247, 34, 16);
+  border-color: rgb(247, 34, 16);
+}
+
+.text-orange {
+  color: rgb(247, 34, 16);
+}
+
+.in-img {
+  margin-top: -30px;
+}
+
+.icon {
+  background: url(https://g.newsweek.com/www/images/logo-n1.svg) center/contain no-repeat #f72210;
+  width: 29px;
+  height: 28px;
+  margin-right: 3px;
+}
+
+.f {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+}
+
+.f2 {
+  font-size: 14px;
+  margin-left: 5px;
+}
+
+.align {
+  position: relative;
+  top: -210px;
+  right: 15px;
+}
+
+.bg-litegray {
+  background-color: rgb(244, 245, 246);
+  height: fit-content;
+}
+
+.bottom-img {
+  height: 240px;
+}
+
+.input {
+  width: 140px;
+  margin-bottom: 5px;
+}
+
+.copy-m {
+  margin-top: 20px;
+}
+
+.icon-m {
+  margin-top: 20px;
+  width: 20px;
+  height: 20px;
+}
+
+.m-bottom {
+  margin-bottom: -15px;
+}
+
+.font-bottom {
+  font-size: 12px;
+}
+
+.m-features {
+  margin-right: 0;
+  margin-left: 0;
+  margin-top: -9px;
+}
+
+.m-top-img {
+  margin-top: 20px;
+}
+
+.right-sec {
+  width: 250px;
+  margin: 50px 10px 10px 0;
+}
+
+.nav-btn {
+  font-size: 16px;
+  width: 250px;
+  background-color: #ffe400;
+  border: none;
+  border-radius: 20px;
+}
+
+.nav-info {
+  margin-left: 10px;
+  width: 150px;
+}
+
+.nav-w {
+  margin-top: 35px !important;
+}
+
+.w-img {
+  width: 10%;
+  height: 10%;
+}
+
+.story {
+  width: 100%;
+}
+
+.height-sus {
+  height: fit-content;
+}
+
+.footer-icon {
+  width: 236px;
+  height: 60px;
+}
+
+.suscribe-1 {
+  background-image: url("https://d.newsweek.com/en/full/1610195/cover-july-24-2020.jpg?w=170&q=95&f=2ea70c30e8f5467c1e61021ae9778138");
+  background-size: cover;
+  width: 170px;
+  height: 220px;
+}
+
+.suscribe-2 {
+  background-image: url("https://d.newsweek.com/en/full/1611327/cover-july-31-august-07-2020.jpg?w=170&q=95&f=e9849b39e78f364f0f8bb410954aa063");
+  background-size: cover;
+  width: 170px;
+  height: 220px;
+  position: relative;
+  top: 0;
+  right: 94px;
+}
+
+.nav-md {
+  display: none;
+}
+
+.nav-sm {
+  display: none;
+}
```

Comment:
Group selectors with the same style definition(same as the previous TSE requested)
You can easily do this by:
``` 
.dis-none {
  display: none;
}
```

And you can use **dis-none** class name in any element you want.

Moreover, you have a built-in Bootstrap class name for this, d-none

Check this life-changing [link](https://getbootstrap.com/docs/4.5/utilities/display/#hiding-elements)